### PR TITLE
sztp: document sztp configuration steps by admin

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

File: ZTP.md [hunk 1]
```diff
@@ -60,8 +60,24 @@ Secure Zero Touch Provisioning (SZTP) adds a bootstrap server to DHCP-based ZTP
 - Both IPv4 and IPv6 DHCP discovery and ZTP provisioning should be supported.
 - For environments that are predominately IPv6 based, we'll need a solution that supports SLAAC.
 - ZTP process and results, both success and failures, must be logged.
+- DHCP is untrusted, only used to point to Bootstrap server.
+- Device must trust network and Network must trust device.
```
Comment (mestery) [inline ZTP.md:64]: Either capitalize both instances of `network` or leave them both without capitals.

File: ZTP.md [hunk 2]
```diff
@@ -60,8 +60,24 @@ Secure Zero Touch Provisioning (SZTP) adds a bootstrap server to DHCP-based ZTP
...
+- Need Image authenticity in addition to onboarding-info authenticity.
```
Comment (mestery) [inline ZTP.md:65]: Maybe don't capitalize `Image` here either.

**Review by mestery — APPROVED** [review]

File: ZTP.md (hunk 1 above)
Comment (glimchb) [reply ZTP.md:64]: on it

File: ZTP.md (hunk 2 above)
Comment (glimchb) [reply ZTP.md]: done

**Pull request merged**